packages/Checkbox/stories/examples/CheckboxExample.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Rule } from "storybook/assets/styles/common.styles";
+import Checkbox from "../../src";
+
+const CheckboxExample = props => {
+  const state1 = React.useState(props.value || true);
+  const state2 = React.useState(props.value || false);
+
+  const handleChange = state => () => {
+    const [, setIsChecked] = state;
+    setIsChecked(prevChecked => !prevChecked);
+  };
+
+  return (
+    <div>
+      <Checkbox {...props} onChange={handleChange(state1)} isChecked={state1[0]}>
+        Authentic VHS beard.
+      </Checkbox>
+      <Rule />
+      <div
+        css={`
+          [data-pka-anchor="checkbox"] {
+            margin-bottom: 12px;
+          }
+        `}
+      >
+        <Checkbox {...props} onChange={handleChange(state2)} isChecked={state2[0]}>
```

Comment:
@mikrotron Are you addressing these stories?

![image](https://user-images.githubusercontent.com/4040102/66081679-55ee3e80-e51d-11e9-8f96-965d82de9cca.png)